src/types/@azure/AiDriver.d.ts

```diff
@@ -0,0 +1,289 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
💡 rename this to more normal type and remove `@azure` from the folder structure? what do you think @kejatura-dev + @hsubramanianaks

Reply:
@azure was just the name of the team/service. But I am okay with changing the name of the folder